src/net_processing.cpp

```diff
@@ -2426,9 +2426,7 @@ void PeerLogicValidation::ProcessMessage(CNode& pfrom, const std::string& msg_ty
         UpdatePreferredDownload(pfrom, State(pfrom.GetId()));
         }
 
-        if (!pfrom.IsInboundConn() && pfrom.IsAddrRelayPeer())
-        {
-            // Advertise our address
```

Comment:
nit: Why removing the comment? :)

Reply:
Sort of justified [here](https://github.com/bitcoin/bitcoin/pull/19724#discussion_r474387266), but i'm not convinced it's the right approach. See my comment there.

Reply:
the comment removal was leftover from a previous iteration of this code where I updated the conditional  to call a function `AdvertiseAddressToConn` or something that made the comment feel redundant. 

latest push includes the more in depth commenting proposed in that thread, will continue convo there